src/components/common/Select.ts

```diff
@@ -290,11 +290,10 @@ const VirtualizedMenuList = (props) => {
   const rowHeight = 40;
   const height = _.clamp(rowHeight, maxHeight, rowHeight * rowCount);
 
-  // @ts-expect-error
   return h(AutoSizer, { disableHeight: true }, [
     ({ width }) => {
+      // @ts-ignore
```

Comment:
Why is this change necessary?

Reply:
The `@ts-expect-error` directives needed to be removed because, with the `@types/react-virtualized` module added, they no longer produced an error. But now this line flags an error, not sure exactly why. I can change this directive to `@ts-expect-error` so it becomes an error if it's not needed at some point.